proto/devices.proto

```diff
@@ -32,12 +32,15 @@ message Device {
   string owner_email = 12;
   string owner_provider = 13;
   string preshared_key = 14;
+  int32 persistent_keepalive_interval = 15;
```

Comment:
Any reason to not echo back the device endpoint here?

Reply:
the endpoint was already being exposed on the device